src/pybind/mgr/volumes/fs/operations/versions/subvolume_base.py

```diff
@@ -203,6 +203,11 @@ def get_attrs(self, pathname):
         except EarmarkException:
             attrs["earmark"] = ''
 
+        try:
+            attrs["encoding"] = self.fs.getxattr(pathname, 'ceph.dir.encoding').decode('utf-8')
```

Comment:
> pybind/mgr/volumes: add encoding to attrs/info
> This was skipped for expediency but it's easy to add.

This seems very casual and it seems like if we're adding something to the interface, it needs to be documented?

Reply:
casual?

If it's alright with you, I'll add the doc to #62904 which is based on this PR.